data/database.py

```diff
@@ -827,6 +833,13 @@ class RepositorySearchScore(BaseModel):
     last_updated = DateTimeField(null=True)
 
 
+class DeletedRepository(BaseModel):
```

Comment:
Actually, not in this case. Here, I want an explicit tracking for only deleted repositories, similar to our tracking for deleted namespaces. Otherwise, we'd need a bunch of new fields which would be redundant 99% of the time.